api/src/middleware/respond.ts

```diff
@@ -23,7 +23,8 @@ export const respond: RequestHandler = asyncHandler(async (req, res) => {
 	}
 
 	if (
-		req.method.toLowerCase() === 'get' &&
+		req.method.toLowerCase() !== 'get' &&
+		req.path?.startsWith('/graphql') === false &&
```

Comment:
I think there is a logical error in this `if`.

This `if` is conjunction of conditions that needs to happen to cache response, so I suppose it should be:
```ts
(req.method.toLowerCase() === 'get' || req.path?.startsWith('/graphql')) &&
env.CACHE_ENABLED === true &&
cache &&
!req.sanitizedQuery.export &&
res.locals.cache !== false &&
exceedsMaxSize === false
```